clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -97,6 +95,51 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:complexType name="cluster-connection-spec">
+    <xs:sequence maxOccurs="1" minOccurs="1">
+      <xs:element name="server" type="tc:server" maxOccurs="unbounded" minOccurs="1"/>
```

Comment:
minOccurs=1 is the default so it's not needed.

Reply:
Indeed. Done.